core/src/toga/widgets/imageview.py

```diff
@@ -63,7 +63,7 @@ def rehint_imageview(image, style, scale=1):
 class ImageView(Widget):
     def __init__(
         self,
-        image: Image | Path | str | None = None,
+        image: Any = None,
```

Comment:
While `Any` isn't *wrong* here, it's not especially helpful; if we're not going to provide an explicit type (`str|Path|Image|PIL_Image|None`....), the text description should clarify that it's not truly an "Any" free-for-all - it should be one of the supported Image constructor types.

Reply:
I think I was trying to anticipate the future plugin situation; I don't know if it'll even be possible to specify the registered types for static type-checking. But yeah, good idea to list them at least while we can, and document everrything.